dapp-oeth/src/components/GetOUSD.js

```diff
@@ -109,6 +109,11 @@ const GetOUSD = ({ id, buttonText, className, style }) => {
 
     if (!userAlreadyConnectedWallet && active) {
       localStorage.setItem('userConnectedWallet', 'true')
+      event({
+        event: 'connect',
+        connect_address: address?.slice(2),
```

Comment:
Any reason to take off the `0x`, does it look at is a hex number if you dont?

Reply:
Yeah, GA converts this into a number automatically 🥲